Kodelock.py

```diff
@@ -1,64 +1,64 @@
+#!/bin/python3
+
+from pathlib import Path
+
 from tkinter import *
-from PIL import ImageTk, Image 
+from PIL import ImageTk, Image
 from pynput import keyboard
 
+
 # Placeholder & Files
-app = Tk()
-codes = open("codes.txt", "r")
-CodeFace = Image.open("./imgs/Face.jpg")
-LeftArrow = Image.open("./imgs/LeftArr.png")
+codes = "codes.txt"
+code_list = Path(codes).read_text().splitlines()
+
+CodeFace  = Image.open("./imgs/Face.jpg")
+LeftArrow  = Image.open("./imgs/LeftArr.png")
 RightArrow = Image.open("./imgs//RightArr.png")
 
 # Settings
+app = Tk()

```

Comment:
Move 'app' into Settings IOT keep like variables in the same location and make it a bit easier to find.